### DISCUSSION TITLE
My student pack application keeps getting declined

### DISCUSSION THREAD

**Discussion by gacanepa** [community/community#69859] [GitHub Education]
### Hi everyone,

I am an online international student at Brigham Young University-Idaho. The university's campus is located in Rexburg, but I reside in Argentina. Even though my account is associated with a valid, current university email, my education pack access is expiring in 11 days and I'm unable to submit proof of my current academic status to renew it due to the following:

- My student portal allows me to download an unofficial transcript that shows my current enrollment (Fall 2023 semester) but the application only accepts an image. 
- The education site accessed my location with my permission. The application process complains that the university does not have a campus where I live. That is why I am an online student.
- I am happy to submit the unofficial transcript (which shows my name and the university's), payment receipts, etc.

If have any ideas, they will be much appreciated.

**Comment by saurabhm02** [comment]
bro just Make sure that you are using the latest version of the education site's application.
make sure that the educational is not used twice 
If you are still having problems, you can contact the education site's support team for assistance.

**Reply by gacanepa** [reply]
@saurabhm02 thank you for your reply. However:
- Where (link?) Is the latest version of the application?
- What do you mean by "make sure the educational is not used twice"? If you are referring to my university email, no - it is not being used twice.
- How do I reach the site support team? The official docs and the former site redirect here.

**Reply by saurabhm02** [reply]
i guess your educational email is verified as bcoz in your profile is show PRO tag it means you r verified

**Reply by saurabhm02** [reply]
However,
https://education.github.com/pack 
this is the where u can register for educational benfits

**Comment by saurabhm02** [comment]
if my suggestion are helping u , please mark as answer

**Reply by gacanepa** [reply]
I appreciate your time, but your replies did not resolve my question.
- I know I am currently verified. But that will change in 11 days unless I can submit proof of my current academic status, like I said in my initial post.
- That site is the same as always. It is where I submitted the application that was declined twice.

**Comment by fullstop125** [comment]
@gacanepa do you by chance have a student ID? That could probably speed up the process, You could also try to use your acceptance letter to the university, all you will do is convert it into an image that clearly shows the date.
